packages/ap-sports/src/lib/schedule.ts

```diff
@@ -0,0 +1,51 @@
+import { datetime, errors } from '@linkpool/shared'
+
+import { statusToStatusId } from './constants'
+
+import type { Game, GameCreate, GameResolve } from './types'
+
+export function getGameCreate(game: Game): GameCreate {
+  const { home, away } = game
+  const gameCreate = {
+    gameId: game.id.replace(/-/g, ''),
+    startTime: datetime.iso8061ToTimestamp(game.scheduled),
+    homeTeam: `${home.market} ${home.name}`,
+    awayTeam: `${away.market} ${away.name}`,
+  }
+  Object.entries(gameCreate).forEach(([key, value]) => {
+    value ??
+      errors.throwError(
+        `'${key}' is null or undefined: ${JSON.stringify(gameCreate)}. Game: ${JSON.stringify(
+          game,
+        )}`,
+      )
+  })
+  return gameCreate
+}
+
+export function getGameResolve(game: Game): GameResolve {
+  const { home, away } = game
+  const gameResolve = {
+    gameId: game.id.replace(/-/g, ''),
+    homeScore: home.runs,
+    awayScore: away.runs,
+    status: statusToStatusId.get(game.status) as number,
+  }
+  Object.entries(gameResolve).forEach(([key, value]) => {
+    value ??
+      errors.throwError(
+        `'${key}' is null or undefined: ${JSON.stringify(gameResolve)}. Game: ${JSON.stringify(
+          game,
+        )}`,
+      )
+  })
+  return gameResolve
+}
+
+export function stringToHex(id: string): string {
```

Comment:
1. Please, be accurate and meaningful with the names. `stringToHex`... converts a string to hex... any string? What is an hex? Why then the validations?

```typescript
export function uuidToBytes32(uuid: string): string {
  const bytes = Buffer.from(uuid.replace(/-/g, ''))
  if (bytes.length === 32) {
    return `0x${bytes.toString('hex')}`
  }
  throw new Error(`Invalid 'uuid': ${uuid}. The uuid is not 32 bytes long`)
}
```
Also let's place it in `shared/src/uuid.ts` (and test it)